openlibrary/plugins/upstream/code.py

```diff
@@ -191,7 +191,7 @@ def POST(self, key):
 
         user = accounts.get_current_user()
         is_admin = user and user.key in [m.key for m in web.ctx.site.get('/usergroup/admin').members]
```

Comment:
I think this line can be removed now, and use the standard method `user.is_admin()` below on at line 194.

Other than this small (optional?) change, this PR looks good to me!

Reply:
Thanks, just did.

Reply:
Just did.